portlayer/attach/messages.go

```diff
@@ -14,17 +14,100 @@
 
 package attach
 
+import "golang.org/x/crypto/ssh"
+
 // All of the messages passed over the ssh channel/global mux are (or will be)
 // defined here.
 
-const (
-	WindowChangeReq = "window-change"
-)
+type Messager interface {
```

Comment:
You don't like back-rubs?  :laughing: 

https://golang.org/doc/effective_go.html#interface-names

It's an interface for sending messages.  So messagER seems right to me.